linkstack/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.8"
+
+services:
+
+  linkstack:
+    hostname: 'linkstack'
+    image: 'linkstackorg/linkstack:latest@sha256:abd691b4293b020a317de8794737671e0315159efcb868e8a4124d6f0611f7ae'
+    environment:
+      TZ: 'Europe/Berlin'
+      SERVER_ADMIN: 
+      HTTP_SERVER_NAME: umbrel.local
+      HTTPS_SERVER_NAME:
+      LOG_LEVEL: 'info'
+      PHP_MEMORY_LIMIT: '256M'
+      UPLOAD_MAX_FILESIZE: '8M'
+    volumes:
+      - 'linkstack_data:/htdocs'

```

Comment:
The data should be stored in the custom umbrelOS app data directory like this:
```suggestion
      - '${APP_DATA_DIR}/data:/htdocs'
```
While testing the app, I cam across the same issue as you, the built-in apache http server seems to show just the default page for a freshly set-up http server. This one is weird, because a new and empty volume "should" be the same as a new and empty bind-mounted directory. @nmfretz do you have some ideas? Ideally we should get rid of the named volume.

Reply:
Ah, darn I just looked into it and this is mentioned in the LinkStack docs:
https://docs.linkstack.org/docker/setup/#docker-bind-mounts

<img width="852" alt="image" src="https://github.com/user-attachments/assets/ab78cf67-1db6-4e4b-b55d-cdea52756f9c">

That's quite the pain, and a really strange requirement. If we're going to bring this to the official app store we should see if we can find a workaround to this.

Reply:
I forgot to post this follow up last week:

If we define a volume like this

```yaml
volumes:
  linkstack_data:
    driver: local
    driver_opts:
      o: bind
      type: none
      device: ${APP_DATA_DIR}/data
```

And then use `linkstack_data`  in the linkstack service like this:

```yaml
volumes:
  - 'linkstack_data:/htdocs'
```

Then we get the syntax of a named volume, but it's actually points to a directory on the host at the location we defined: `${APP_DATA_DIR}/data`

Doing it this way seems to work. I'm just not quite sure if there are any downsides to this that I haven't thought of.